README.md

```diff
@@ -41,9 +41,34 @@ are package private and are not part of the public API.
   * [AccessExpression](src/main/java/org/apache/accumulo/access/AccessExpression.java).
   * [Authorizations](src/main/java/org/apache/accumulo/access/Authorizations.java).
 
+## Getting Started
+
+Add the library to your CLASSPATH. For Maven, use:
+
+```xml
+<dependencies>
+  <dependency>
+    <groupId>org.apache.accumulo</groupId>
+    <artifactId>accumulo-access</artifactId>
+    <version>$version</version>
+  </dependency>
+</dependencies>
+```
+
+## Running the Example
```

Comment:
This shows how to run it, but does not show how to run it with different auths to get different output. See the "Example runs" section of the old README.

Reply:
I converted that into behavior in the example code itself. If you run it this way, it tells you that you can provide the auths to see a specific example, and runs it through the examples from the old README. I can tweak the message, or add an extra line to the README to make it more clear. Which would be better?

Reply:
I did some clarification in 48e7c2836ab4560eb449f5f1e5269bfb6961924b